### PR TITLE
Explore split + Data update

### PR DESCRIPTION
**Pull request by LRWeber**
Main `explore` directory is now split into two directories:

- `explore`
Still contains the `github-data` files and webpage.

- `_explore`
Contains all of the data collection scripts and additional contents that don't need to be packaged into the website.

Data collection scripts now utilize a `GITHUB_DATA` environment variable. This will greatly simplify any future relocation of our JSON files.

Also included a bug fix for the License word cloud. ("Other" was not properly excluded, which heavily skewed the counts and word scaling.)

### CODE REVIEW

File: _explore/scripts/build_yearlist.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
 from scraper.github import queryManager as qm
+from os import environ as env
 import os.path
 
 yearDict = {}
 
 # Gather all file name data
 print("Checking GitHub data file names with year stamps...")
-for file in os.listdir("../github-data"):
+for file in os.listdir(env['GITHUB_DATA']):
```
Comment (IanLee1521) [inline _explore/scripts/build_yearlist.py]: You might want to consider doing something like `env.get('GITHUB_DATA', '../github-data')` or similar to set a default base directory in the event the environment variable is missing.

File: explore/github-data/LAST_MASTER_UPDATE.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-2018-11-06
+2018-11-13
```
Comment (IanLee1521) [inline explore/github-data/LAST_MASTER_UPDATE.txt]: In the future, can we leave the data updates as separate pull requests? It makes the reviewing and merging easier. :)

File: _explore/scripts/build_yearlist.py (hunk 1 above)
Comment (LRWeber) [reply _explore/scripts/build_yearlist.py]: Got it, adding via #120

File: explore/github-data/LAST_MASTER_UPDATE.txt (hunk 2 above)
Comment (LRWeber) [reply explore/github-data/LAST_MASTER_UPDATE.txt]: Whoops, yes, this one should have been done separately.

What do you suggest for other pull requests that are dependent on changes to the data? E.g. #118 , many of the new elements on that branch rely on additional pieces of data that were previously not included, thus requiring a complete update to function.

**Pull request merged**